stages/org.osbuild.systemd.unit

```diff
@@ -20,6 +20,10 @@ def main(tree, options):
         unit_dropins_dir = f"{tree}/usr/lib/systemd/system/{unit}.d"
     elif unit_type == "global":
         unit_dropins_dir = f"{tree}/usr/lib/systemd/user/{unit}.d"
+    else:
+        print(f"Error: Unknown unit type: {unit_type}")
```

Comment:
(nitpick) for consistency this could also raise an exception like ValueError (we do it a few lines above)  [but not very important]

Reply:
It does and it's fine for the PR. I think it shows the deeper issue (that is much bigger than this PR)  that consistency for the error behavior in osbuild stages is not great. I personally find it not ideal that we have a powerful language like python and then do error handling in the bash style of return code where "0" mean "good" instead of "false" :) But sorry, your reasoning is sound of course, this is a different problem from this PR.